src/index.html

```diff
@@ -1,13 +1,45 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+        src="./images/imac.jpeg" 
+        alt="screen" 
+        class="card__img"
+      >
+      <div class="card__name-item">
+      APPLE A1419 iMac 27&#34; Retina 5K Monoblock &#40;MNED2UA&#47;A&#41;
+      </div>
+      <div class="card__name-article">Product code&#58; 195434</div>
```

Comment:
'p'

Reply:
please use `p` there as it requires in the [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) p 12